functions/firebase_rtdb/test/DeployTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace Google\Cloud\Samples\Functions\HelloworldStorage\Test;
+
+use Google\Cloud\Logging\LoggingClient;
+use Google\Cloud\TestUtils\CloudFunctionDeploymentTrait;
+use Google\Cloud\TestUtils\EventuallyConsistentTestTrait;
+use Google\Cloud\TestUtils\GcloudWrapper\CloudFunction;
+use GuzzleHttp\Client;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\ExpectationFailedException;
+
+/**
+ * Class DeployTest.
+ *
+ * This test is not run by the CI system.
+ *
+ * To skip deployment of a new function, run with "GOOGLE_SKIP_DEPLOYMENT=true".
+ * To skip deletion of the tested function, run with "GOOGLE_KEEP_DEPLOYMENT=true".
+ */
+class DeployTest extends TestCase
+{
+    use CloudFunctionDeploymentTrait;
+    use EventuallyConsistentTestTrait;
+
+    /** @var string */
+    private static $entryPoint = 'firebaseRTDB';
+    
+    /** @var string */
+    private static $functionSignatureType = 'cloudevent';
+
+    /** @var string */
+    private static $rtdbPath = 'foods';
+
+    /** @var LoggingClient */
+    private static $loggingClient;
+
+    /** @var Database */
+    private static $database;
+
+    /**
+     * Deploy the Cloud Function, called from DeploymentTrait::deployApp().
+     *
+     * Overrides CloudFunctionDeploymentTrait::doDeploy().
+     */
+    private static function doDeploy()
+    {
+        $project = self::requireEnv('GOOGLE_PROJECT_ID');
+
+        $resource =
+            'projects/_/instances/' . $project . '/refs/' . self::$rtdbPath;
```

Comment:
Consider using `sprintf` for string building
```suggestion
           sprintf('projects/_/instances/%s/refs/%s', $project, self::$rtdbPath);
```

Reply:
Done.